ReflectionAnalyzers.Vsix/source.extension.vsixmanifest

```diff
@@ -1,30 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="ReflectionAnalyzers.Vsix.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="1.0" Language="en-US" Publisher="Johan Larsson" />
-    <DisplayName>ReflectionAnalyzers.Vsix</DisplayName>
+    <Identity Id="ReflectionAnalyzers.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="1.0" Language="en-US" Publisher="Johan Larsson" />
```

Comment:
We could maybe keep the version synced with the nuget? And try to remember it.

Reply:
Hold my beer.

Reply:
Going to force push a bunch of test commits probably

Reply:
Push like you mean it!

Reply:
I've set it up so that running `build.ps1` first sets the VSIX version to the analyzer project version prefix. What do you think?

Reply:
Sounds good! I'll try to remember to run it or update manually.